spec/unit/pushprocessor.spec.ts

```diff
@@ -712,6 +713,37 @@ describe("NotificationService", function () {
             });
         });
     });
+
+    describe("test intentional mentions behaviour", () => {
+        it.each([RuleId.ContainsUserName, RuleId.ContainsDisplayName, RuleId.AtRoomNotification])(
+            "disable legacy rule: %s",
```

Comment:
Can we find a fuller description? It took me a while to work out what this test does.

How about "Rule %s matches in legacy mode and does not in intentional mentions mode"

(Or something better...)